src/lib/gossip.ts

```diff
@@ -80,6 +80,13 @@ export class Gossip {
     this.players = gossip.players || [];
     this.requestingCompost = gossip.requestingCompost || [];
     this.stench = gossip.stench || 0;
+    const heapStench = parseInt(get(CURRENT_STENCH)) || 0;
+    if (heapStench !== this.stench) {
```

Comment:
Will this ever agree?

Reply:
probably not. It was nice for early debugging but I'll take it out